client/python/ovmsclient/lib/ovmsclient/tfs_compat/http/serving_client.py

```diff
@@ -56,6 +58,10 @@ def predict(self, inputs, model_name, model_version=0, timeout=10.0):
         except ModelServerError as model_server_error:
             raise model_server_error
         except (JSONDecodeError, ValueError) as parsing_error:
+            if raw_response.status_code is not HTTPStatus.OK:
+                error_code = HTTPStatus(raw_response.status_code)
+                error_message = f"{error_code.value} {error_code.phrase}"
+                raise_from_http_response(error_code, error_message)
```

Comment:
I suppose that this solves the issue with too big http request. I wonder if there could be better solution than trying to parse the response to json and then after it fails validating if this response was even valid.

Reply:
Yes, I think this is something we could refactor to have cleaner flow, but I'd do this for 2022.2 release.